lib/blockchain_test.go

```diff
@@ -741,10 +741,27 @@ func _signTxn(t *testing.T, txn *MsgBitCloutTxn, privKeyStrArg string) {
 	privKeyBytes, _, err := Base58CheckDecode(privKeyStrArg)
 	require.NoError(err)
 	privKey, _ := btcec.PrivKeyFromBytes(btcec.S256(), privKeyBytes)
-	require.NoError(err)
 	txnSignature, err := txn.Sign(privKey)
 	require.NoError(err)
-	txn.Signature = txnSignature
+	txn.Signature = txnSignature.Serialize()
+}
+
+// Signs a txn with derived key. Signature contains solution iteration,
```

Comment:
I think we can delete this now. Sad; it was a fun function.

Reply:
This one actually comes in handy in `block_view_test.go`, I updated it now to reflect the new `ExtraData` approach.